lib/skate_web/templates/layout/app.html.heex

```diff
@@ -51,7 +47,7 @@
     <main role="main" class="container">
       <%= @inner_content %>
     </main>
-    <script type="text/javascript" src="<%= static_href.(@conn, "/js/app.js") %>"></script>
-    <%= _drift(assigns) %>
+    <script type="text/javascript" src={~p"/js/app.js"}></script>
```

Comment:
**question:** on this and other places where static paths are replaced - does this still get turned into a URL pointing to cdn.mbta.com rather than skate.mbta.com?

Reply:
I bet it does not. I'll probably have to return these to their previous forms. Will check in dev-blue.